packages/cli/src/runDeploy.ts

```diff
@@ -95,8 +95,7 @@ in your contracts directory to use the default anvil private key.`,
 
   const resolvedConfig = resolveConfig({ config, forgeSourceDir: srcDir, forgeOutDir: outDir });
 
-  const keyId = opts.awsKmsKeyId ?? process.env.AWS_KMS_KEY_ID;
-  const account = keyId ? await kmsKeyToAccount({ keyId }) : privateKeyToAccount(privateKey);
+  const account = opts.aws ? await kmsKeyToAccount({}) : privateKeyToAccount(privateKey);
```

Comment:
we could also default the options object to `{}` so we don't need to pass an empty object
```suggestion
  const account = opts.aws ? await kmsKeyToAccount() : privateKeyToAccount(privateKey);
```